app/src/pages/income.tsx

```diff
@@ -0,0 +1,88 @@
+import incomeData from '@public/data/income.json'
+import type { GetServerSideProps, NextPage } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { ChangeEvent, useState } from 'react'
+import styled from 'styled-components'
+
+import ButtonLink from '@components/ButtonLink'
+import Dropdown from '@components/Dropdown'
+
+const Income: NextPage = () => {
+  const { t } = useTranslation('common')
+  const [householdSize, setHouseholdSize] = useState<keyof typeof incomeData>()
+  // @ts-ignore TODO: PAIR WITH ROCKET ON THIS
+  const householdSizes: keyof typeof incomeData[] = Object.keys(incomeData)
+
+  const handleChange = (e: ChangeEvent<HTMLSelectElement> & { target: { value: keyof typeof incomeData }}) => {
+    setHouseholdSize(e.target.value) 
+  }
+
+  return (
+    <>
+      <h1>{t('Income.title')}</h1>
+      <p>{t('Income.enrolled')}</p>
+      <p>{t('Income.notEnrolled')}</p>
+      <Dropdown
+        id="income"
+        label={t('Income.dropdownLabel')}
+        // @ts-ignore
+        handleChange={handleChange}
+        // @ts-ignore
+        options={householdSizes}
+      />
+      <Helper>{t('Income.helper')}</Helper>
+      <Table className="usa-table usa-table--borderless">
+        <caption>
+          <h2>{t('Income.estimatedIncome')}</h2>
+        </caption>
+        <thead>
+          <tr>
+            <th scope="col">Annual</th>
+            <th scope="col">Monthly</th>
+            <th scope="col">Weekly</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">{householdSize && incomeData[householdSize]?.annual || '-'}</th>
+            <td>{householdSize && incomeData[householdSize]?.monthly || '-'}</td>
+            <td>{householdSize && incomeData[householdSize]?.weekly || '-'}</td>
+          </tr>
+        </tbody>
+      </Table>
```

Comment:
I like that you structured this as a table, but the mobile styling doesn't match the mockups.

Reply:
yeah I guess I was still going off of the outdated figma -- I was planning on polishing all pages with the ticket updated design styling. But I can update this in this PR

Reply:
Ah I see. Let's stick to your plan!